download.tt

```diff
@@ -6,60 +6,55 @@
-    The latest stable release is <strong>[%latestNixVersion%]</strong>.
-  </p>
+<section class="download-header">
+  <div>
+    <h1>Nix</h1>
+    <div>
+      <h2>Current version</h2>
+      <a href="#">[% latestNixVersion %]</a>
+    </div>
+  </div>
 </section>
 
-<section class="generic-layout">
-  <h1>Getting Nix</h1>
-  <div class="tabs-navigation">
-    <nav>
-      <ul id="getting-nix">
-        <li><a href="#nix-quick-install">Quick install</a></li>
-        <li><a href="#nix-verify-installation">Verify installation (GPG)</a></li>
-        <li><a href="#nix-uninstall">Uninstall</a></li>
-        <li><a href="#nix-more">More ...</a></li>
-      </ul>
-    </nav>
-
+<section class="download-nix">
+  <div class="collapse">
     <div>
-      <section id="nix-quick-install">
-        The quickest way to install Nix is to open a terminal and run the following
-        command (as a user other than <code>root</code> with <code>sudo</code> permission):
+      <article>
+        <h2><span>1.</span>Quick install</h2>
+        <p>
+          The quickest way to install Nix is to open a terminal and run the following
+          command (as a user other than <code>root</code> with <code>sudo</code> permission):
+        </p>
         <pre class="terminal-console">
 <span class="shell-prompt">$ </span>curl -L https://nixos.org/nix/install | sh</pre>
         <p>Make sure to follow the instructions output by the script.</p>
         <p>The installation script requires that you have <code>sudo</code> access to <code>root</code>.</p>
-      </section>
-
-      <section id="nix-verify-installation">
+      </article>
+      <article>
+        <h2><span>2.</span>Verify installation</h2>
         <p>You may want to verify the integrity of the installation script using GPG:</p>
         <pre class="terminal-console">
-<span class="shell-prompt">$ </span>curl -o install-nix-[%latestNixVersion%] https://releases.nixos.org/nix/nix-[%latestNixVersion%]/install
-<span class="shell-prompt">$ </span>curl -o install-nix-[%latestNixVersion%].asc https://releases.nixos.org/nix/nix-[%latestNixVersion%]/install.asc
-<span class="shell-prompt">$ </span>gpg2 --recv-keys B541D55301270E0BCF15CA5D8170B4726D7198DE
-<span class="shell-prompt">$ </span>gpg2 --verify ./install-nix-[%latestNixVersion%].asc
-<span class="shell-prompt">$ </span>sh ./install-nix-[%latestNixVersion%]</pre>
+<span cls="shell-prompt">$ </span>curl -o install-nix-[%latestNixVersion%] https://releases.nixos.org/nix/nix-[%latestNixVersion%]/install
+<span cls="shell-prompt">$ </span>curl -o install-nix-[%latestNixVersion%].asc https://releases.nixos.org/nix/nix-[%latestNixVersion%]/install.asc
+<span cls="shell-prompt">$ </span>gpg2 --recv-keys B541D55301270E0BCF15CA5D8170B4726D7198DE
+<span cls="shell-prompt">$ </span>gpg2 --verify ./install-nix-[%latestNixVersion%].asc
+<span cls="shell-prompt">$ </span>sh ./install-nix-[%latestNixVersion%]</pre>
```

Comment:
`s/cls/class/` ??

Reply:
Why? This would break a rule I follow when writing: content should be in the file, and not in the CSS. This is part of the content here.

If it's about being able to select the content, we have the `#no-select();` LESS mixin which could be used here. We might want to apply `#no-select();` to something like `pre .shell-prompt { #no-select(); }`